backend/backend.go

```diff
@@ -518,9 +518,16 @@ func (b *Backend) runOnce() error {
 
 	_ = b.Client.Close()
 
+	eCloser := b.EventStore.(closer)
+	_ = eCloser.Close()
```

Comment:
Would defer make sense here (in the event more code is added)?

Reply:
Hum, yes it would!